pkg/agent/multicast/mcast_controller_test.go

```diff
@@ -144,6 +149,72 @@ func TestUpdateGroupMemberStatus(t *testing.T) {
 	}
 }
 
+func TestCheckNodeUpdate(t *testing.T) {
+	mockController := newMockMulticastController(t, false)
+	err := mockController.initialize(t)
+	assert.Nil(t, err)
+
+	for _, tc := range []struct {
+		oldNode     *corev1.Node
```

Comment:
Add "name" property in the test case struct, and use this format for each case?

```
t.Run(tc.name, func (t *testing.T) {
	mockController.nodeUpdateQueue = workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(minRetryDelay, maxRetryDelay), "nodeUpdate")
	mockController.checkNodeUpdate(tc.oldNode, tc.curNode)
	if tc.nodeUpdated {
		assert.Equal(t, 1, mockController.nodeUpdateQueue.Len())
	} else {
		assert.Zero(t, mockController.nodeUpdateQueue.Len())
	}
})
```

Reply:
Updated all tests in multicast module. For multiple cases test, I changed it with `.Run(tc.name, func (t *testing.T) {` and for single case test, the for loop is removed.